src/containers/MainForm/EventOptions.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import {
+  Select,
+} from 'antd';
+const { Option } = Select;
+
+export const renderEventOptions = (mode, state) => {
+  let lawmakerType;
+  if (mode === 'moc' && state === null) {
+    lawmakerType = 'fedRep';
+  } else if (mode === 'moc' && state !== null) {
```

Comment:
just do a truthy check for state instead of having it equal null

Reply:
updated truthy check